twitter-processor-example/docker/connectors-docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '2'
+services:
+  activiti-cloud-connector-twitter:
+      image: activiti-cloud-connector-twitter
+      environment:
+        - TWEET_PROCESS_DEF=tweet-processor:1:2ef2d39d-d13f-11e7-8988-0242ac140005
```

Comment:
I'm wondering if it's not better to use definition key instead of the process id.